packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -680,6 +731,11 @@ class _DragAvatar<T> extends Drag {
       }
     }
 
+    // Report moves to the targets.
```

Comment:
The event of mouse movement (according to [MouseTracker](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/rendering/mouse_tracking.dart#L543)) is exit, enter, then move. Might as well use the same order here, by moving the move event to after entering?